Gemfile

```diff
@@ -12,4 +12,5 @@ group :development do
   gem 'tee'
   gem 'webmock'
   gem 'yard'
+  gem 'buildpack-packager', git: 'https://github.com/cloudfoundry-incubator/buildpack-packager'
```

Comment:
The Java buildpack doesn't use the packager from other buildpacks.  We use a more sophisticated packager.